be/src/formats/parquet/file_reader.cpp

```diff
@@ -209,22 +209,6 @@ StatusOr<bool> FileReader::_filter_group(const tparquet::RowGroup& row_group) {
             if (discard) {
                 return true;
             }
-
-            // skip topn runtime filter, because it has taken effect on min/max filtering
-            // if row-group contains exactly one value(i.e. min_value = max_value), use bloom filter to test
-            if (!filter->always_true() && min_chunk->columns()[0]->equals(0, *max_chunk->columns()[0], 0)) {
```

Comment:
why remove this?

Reply:
This is done for pruning iceberg table with partition column. Can be superseded by DPP